.circleci/config.yml

```diff
@@ -128,7 +128,7 @@ commands:
     steps:
       - run:
           name: Run JVM unit tests & create jacoco code coverage report
-          no_output_timeout: 30m
+          no_output_timeout: 15m
```

Comment:
Nice, suppose that's unrelated to the flaky tests itself? Maybe make this even smaller to fail faster?

Reply:
yes, it's no related to flaky tests. But we have another issues with tests and wait too long before this timeout